sellercenter-category/src/main/java/com/gfg/sellercenter/category/entity/CategoryWithProductInformation.java

```diff
@@ -0,0 +1,28 @@
+package com.gfg.sellercenter.category.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+import javax.validation.constraints.NotNull;
+
+@AllArgsConstructor
+@Getter
+@EqualsAndHashCode
+public class CategoryWithProductInformation
+{
+    @NotNull
+    private final int idCatalogCategory;
```

Comment:
should this be named the same as in the graph service?

Reply:
Why it should?

Reply:
> should this be named the same as in the graph service?

And even if i would do it i would give the graph service itself already in upfront a better naming instead of just Id ;)